source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -404,11 +433,22 @@ Allowed configuration combinations
 Drivers MUST enforce:
 
 * TopologyType Single cannot be used with multiple seeds.
+* ``directConnection=true`` cannot be used with multiple seeds.
```

Comment:
Isn't this the same as the bullet above? My understanding is that the topology type cannot be Single unless `directConnection` is set to true.